pkgs/rancher-cli.nix

```diff
@@ -0,0 +1,42 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, source
+}:
+
+buildGoModule rec {
+  pname = "rancher-cli";
+  version = source.version;
+
+  src = fetchFromGitHub {
+    owner = source.owner;
+    repo = source.repo;
+    rev = "v${version}";
+    sha256 = source.sha256;
+  };
+
+  vendorSha256 = source.vendorSha256;
+
+  doCheck = false;
+
+  postInstall = ''
+      mv $out/bin/cli $out/bin/rancher
+    '';
+
+  CGO_ENABLED = 0;
+
+  ldflags = ''
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.Version=${source.version}
+      -X github.com/vmware-tanzu/velero/pkg/buildinfo.GitSHA=${source.rev}
+      -X main.VERSION=${source.version}
+      -w -s -extldflags -static
+  '';
```

Comment:
This should be a list: https://nixos.org/manual/nixpkgs/stable/#ssec-go-common-attributes